ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -186,6 +186,7 @@ internal class CustomerCenterViewModelImpl(
     override val actionError: State<PurchasesError?>
         get() = _actionError
     private val _actionError: MutableState<PurchasesError?> = mutableStateOf(null)
+    private var latestCustomerInfo: CustomerInfo? = null
```

Comment:
Hmm I'm not sure if we should cache the CustomerInfo here... Maybe we could return the CustomerInfo as part of the `loadPurchases` method (So return a Pair including the `CustomerInfo` and `PurchaseInformation`) so we can use that information... Otherwise, we could include the required information inside `PurchaseInformation`? Feels like that should be ok.